docs/Initial Setup/InitialSetup.md

```diff
@@ -19,3 +21,14 @@ Launch Visual Studio
 * Open ./BEIMA.Backend/BEIMA.Backend.sln
 * Run the project in Debug Any CPU
 * API is ready to accept requests
+
+Launch Mongo Compass
+ * Click New Connection
+ * Type or paste " mongodb://localhost:27017 " into the connection string textbox
+ * Click Connect
+ * Click the + sign at the bottom of the left pane to add a database
+ * Database name: beima
+ * Collection name: deviceTypes
```

Comment:
Should we add devices to this as well?

Reply:
The initial database creation requires a collection name so I picked the one at the top of the list.
The following steps:

"This will create the DB with one of the required collections.
Once that is completed the + sign inline with the DB name to add the remaining required collections to mirror the production environment."

are there to direct creation of other collections to match the prod environment.

Reply:
Also, the backend should create the collections for you, as long as they are specified in the local.settings.json.